### PR TITLE
Fixing links to wiki articles

### PR DESCRIPTION
**Pull request by ardalis**
## Summary

Fixing links to wiki articles

### CODE REVIEW

File: docs/standard/modernize-with-azure-and-containers/walkthroughs-technical-get-started-overview.md [hunk 1]
```diff
@@ -212,7 +212,7 @@ As main conclusion, using Azure Container Instances is a very compelling option
 
 Explore this content more in-depth on the GitHub wiki:
 
-[https://github.com/dotnet-architecture/eShopModernizing/wiki/05.-Deploying-the-Apps-to-ACI-(Azure-Container-Instances)](https://github.com/dotnet-architecture/eShopModernizing/wiki/05.-Deploying-the-Apps-to-ACI-(Azure-Container-Instances)TBD)
+[https://github.com/dotnet-architecture/eShopModernizing/wiki/05.-Deploying-the-Apps-to-ACI-(Azure-Container-Instances)](https://github.com/dotnet-architecture/eShopModernizing/wiki/05.-Deploying-the-Apps-to-ACI-(Azure-Container-Instances))
 
 ## Walkthrough 5: Deploy your Windows Containers-based apps to Kubernetes in Azure Container Service
```
Comment (nishanil) [inline docs/standard/modernize-with-azure-and-containers/walkthroughs-technical-get-started-overview.md]: The link is broken here.

File: docs/standard/modernize-with-azure-and-containers/walkthroughs-technical-get-started-overview.md [hunk 2]
```diff
@@ -270,9 +270,29 @@ With Kubernetes, developers can progress from thinking about physical and virtua
 
 - Application health checks
 
+## Walkthrough 6: Deploy your Windows Containers-based apps to Azure App Service for Containers
```
Comment (nishanil) [inline docs/standard/modernize-with-azure-and-containers/walkthroughs-technical-get-started-overview.md]: Add Scenario with diagram? Every other walkthrough has it. This is our most recommended path for Windows Containers and we should have that explicitly called out :)

**Review by nishanil — CHANGES_REQUESTED** [review]

File: docs/standard/modernize-with-azure-and-containers/walkthroughs-technical-get-started-overview.md (hunk 1 above)
Comment (nishanil) [reply docs/standard/modernize-with-azure-and-containers/walkthroughs-technical-get-started-overview.md]: In Walkthrough 5. I couldn't comment there..

**Comment by ardalis** [comment]
Hopefully all set now @nishanil

File: docs/standard/modernize-with-azure-and-containers/conclusions.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ ms.date: 10/26/2017
 
 - A Docker container is becoming the standard unit of deployment for any server-based application or service.
 
-- For production environments, you should use an orchestrator (like Kubernetes) to host scalable Windows Containers­­–based applications.
+- For production environments, you should use an orchestrator (like Kubernetes) to host scalable Containers­­–based applications.
```
Comment (rpetrusha) [inline docs/standard/modernize-with-azure-and-containers/conclusions.md]:
```suggestion
- For production environments, you should use an orchestrator (like Kubernetes) to host scalable containers­­–based applications.
```

File: docs/standard/modernize-with-azure-and-containers/index.md [hunk 4]
```diff
@@ -114,7 +114,7 @@ Each maturity level in the modernization process is associated with the followin
...
 During this initial modernization, you can also add assets from the cloud, such as monitoring with tools like [Azure Application Insights](https://docs.microsoft.com/azure/application-insights/app-insights-overview); CI/CD pipelines for your app lifecycles with [Azure DevOps Services](https://azure.microsoft.com/services/devops/); and many more data resource services that are available in Azure. For instance, you can modify a monolithic web app that was originally developed by using traditional [ASP.NET Web Forms](https://www.asp.net/web-forms) or [ASP.NET MVC](https://www.asp.net/mvc), but now you deploy it by using Windows Containers. When you use Windows Containers, you should also migrate your data to a database in [Azure SQL Database Managed Instance](https://docs.microsoft.com/azure/sql-database/), all without changing the core architecture of your application.
```
Comment (rpetrusha) [inline docs/standard/modernize-with-azure-and-containers/index.md]:
```suggestion

During this initial modernization, you can also add assets from the cloud, such as monitoring with tools like [Azure Application Insights](https://docs.microsoft.com/azure/application-insights/app-insights-overview); CI/CD pipelines for your app lifecycles with [Azure DevOps Services](https://azure.microsoft.com/services/devops/); and many more data resource services that are available in Azure. For instance, you can modify a monolithic web app that was originally developed by using traditional [ASP.NET Web Forms](https://www.asp.net/web-forms) or [ASP.NET MVC](https://www.asp.net/mvc), but now you deploy it by using Windows Containers. When you use Windows Containers, you should also migrate your data to a database in [Azure SQL Database Managed Instance](https://docs.microsoft.com/azure/sql-database/), all without changing the core architecture of your application.
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
This looks good, @ardalis. I've made two minor suggestions about formatting and capitalization.

**Comment by ardalis** [comment]
:shipit:

**Comment by rpetrusha** [comment]
Thanks, @ardalis. I'll merge your PR now.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Sorry, @ardalis., @nishanil still has to approve.

**Review by nishanil — APPROVED** [review]
Looks good. Thanks, @ardalis !!

**Pull request merged**